src/components/manage/Widgets/GeolocationWidget.jsx

```diff
@@ -2,11 +2,13 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { Grid } from 'semantic-ui-react';
 import { defineMessages, injectIntl } from 'react-intl';
-import { FormFieldWrapper } from '@plone/volto/components';
+import { FormFieldWrapper, CheckboxWidget } from '@plone/volto/components';
 import { unionBy } from 'lodash';
+import SidebarPopup from '@eeacms/volto-block-style/SidebarPopup/SidebarPopup';
```

Comment:
Please add this dependency to README and DEVELOP

Reply:
Sure.